queries.sql

```diff
@@ -21,5 +21,83 @@ SELECT * FROM animals WHERE neutered = true;
 -- Find all animals not named Gabumon.
 SELECT * FROM animals WHERE NOT name = 'Gabumon';
 
--- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
+/* 
+** Find all animals with a weight between 10.4kg and 17.3kg (including the 
+** animals with the weights that equals precisely 10.4kg or 17.3kg)
+*/
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+-- Start a transaction to set all species to 'unspecified' and roll it back afterwards
+BEGIN;
+UPDATE animals SET species='unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+
+
+-- Start a transaction and persist the data afterwards
+BEGIN:
+/*
+Update the animals table by setting the species column
+** to digimon for all animals that have a name ending in mon
+*/
+UPDATE animals SET species='pigimon' WHERE name LIKE '%mon';
```

Comment:
- [ ] The name of the species should be digimon :smile: 
```suggestion
UPDATE animals SET species='digimon' WHERE name LIKE '%mon';
```

Reply:
Oh! :smile:
Thank you.

Reply:
This change has been made succesfully